.github/workflows/gh-actions-release.yml

```diff
@@ -18,30 +18,44 @@ jobs:
           - FROM:     'ubuntu:focal'
             COMPILER: 'clang'
             FLAGS:    '-DUSE_PYTHON_3=ON'
+            ARTIFACT_EXT: 'deb'
+          # Source tarballs only
+          - FROM:     'ubuntu:focal'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+            ARTIFACT_EXT: 't?z*'
```

Comment:
Might be better to use:
`t[xb]z*` as we don't want to capture tgz files.

Reply:
Done 👍🏽